README.md

```diff
@@ -93,7 +93,16 @@ builder.Services.AddAkka("MyActorSystem", configurationBuilder =>
 })
 ```
 
-#### `ActorRegistry`
+## Dependency Injection Outside and Inside Akka.NET
+
+One of the other design goals of Akka.Hosting is to make the dependency injection experience with Akka.NET as seamless as any other .NET technology. We accomplish this through two new APIs:
+
+* The `ActorRegistry`, a `IServiceCollection`-ish container that is designed to be populated with `Type`s for keys and `IActorRef`s for values _after_ the Akka.NET `IHostedService` and the `ActorSystem` have been started.
```

Comment:
Maybe instead of saying "a `IServiceCollection`-ish" we should say "a dependency injection container for `IActorRef`s, just like `IServiceCollection` is for ASP.NET services"?